src/Views/AssetsManager/AssetsManager.js

```diff
@@ -10,7 +10,9 @@ const THREEUtils = require('../../Game/Components/THREEUtils');
 /**
  * Default material used by native objects
  */
-const DEFAULT_MATERIAL = new THREE.MeshLambertMaterial({ color: 0x00ff00 });
+const DEFAULT_MATERIAL = new THREE.MeshLambertMaterial({
```

Comment:
Why it's a global variable ?

Reply:
to avoid to write "this" before it and because it's not going to change (const) but it could be a member attribute since it's only use in this file